waspc/data/Generator/templates/server/src/config.js

```diff
@@ -8,10 +8,32 @@ const env = process.env.NODE_ENV || 'development'
 //   - Use convict library to define schema and validate env vars.
 //  https://codingsans.com/blog/node-config-best-practices
 
+const defaultOrigins = [
+    'localhost',
```

Comment:
Let' make indentation 2 spaces

Reply:
Got it, will update it later during the day.